get_dependencies.py

```diff
@@ -24,18 +24,40 @@ def download(url, filename):
     sys.stdout.write('\n')
 
 
-print('[*] Extracting webdriver archives...')
-for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
-    tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
-    tar.extractall(path='tools/')
-    tar.close()
+if _platform == "linux" or _platform == "linux2":  # linux
+    osys = "linux"
+elif _platform == "darwin":    # MAC OS X
+    osys = "mac"
+
+
+if osys == "linux":
+    print('linux os detected !')
+    print('[*] Downloading support files for linux ~ 108 mb')
+    download('https://www.dropbox.com/s/3mosdy9tsf7pilh/linux.zip?dl=1', 'linux.zip')
+    print('[*] Extracting linux.zip...')
+    with ZipFile('linux.zip', 'r') as z:
+        z.extractall("linux")
+    print('[*] Extracting webdriver archives...')
+    for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
+        tar = tarfile.open('linux/%s.tar.xz' % f, 'r:xz')
+        tar.extractall(path='tools/')
+        tar.close()
+    os.remove('linux.zip')
+    os.system("rm -rf linux ")
+
+elif osys == "mac":
+    print('mac os detected !')
+    print('[*] Downloading support files for mac ~ 150 mb')
+    download('https://www.dropbox.com/s/k1szymmktj0e1pf/mac.zip?dl=1', 'mac.zip')
+    print('[*] Extracting mac.zip...')
+    os.system("unzip mac.zip -d tools")
```

Comment:
Why aren't you using ZipFile like on Linux?

Reply:
because files get corrupt after extracting it with standard functions of python

Reply:
This seems really strange. I would expect the standard Python functions to be tested on Mac too. Are the files corrupted without any error message or exception thrown?

Reply:
no error message but the drivers i.e geckodriver and chrome driver get converted to text editable and chrome and nightly does not open .

Reply:
Maybe you just need to set them as executable?